x-pack/plugins/security/server/lib/authentication/__tests__/authenticator.js

```diff
@@ -265,8 +270,14 @@ describe('Authenticator', () => {
 
     it('replaces existing session with the one returned by authentication provider.', async () => {
       const user = { username: 'user' };
-      const systemAPIRequest = requestFixture({ headers: { authorization: 'Basic xxx-new' } });
-      const notSystemAPIRequest = requestFixture({ headers: { authorization: 'Basic yyy-new' } });
+      const oldAuth = Buffer.from('foo:notbar').toString('base64');
+      const newAuth = Buffer.from('foo:bar').toString('base64');
+      const systemAPIRequest = requestFixture({ headers: { authorization: `Basic ${oldAuth}` } });
+      const notSystemAPIRequest = requestFixture({ headers: { authorization: `Basic ${oldAuth}` } });
+      const loginAttempt = new LoginAttempt();
+      loginAttempt.setCredentials('foo', 'bar');
```

Comment:
Same question with this test as above.

Reply:
I pushed 0e18d71c454e41867886469ac38b0a7ca9e431bc which should make this test more clear as well.